lib/Resource.js

```diff
@@ -330,6 +330,13 @@ class Resource {
 
 	/**
 	 * Retrieve the project assigned to the resource
+	 * <br/>
+	 * <b>Note for UI5 Tooling extensions (i.e. custom tasks, custom middleware):</b>
+	 * In order to ensure compatibility across UI5 Tooling versions, consider using the
+	 * <code>getProject(resource)</code> methods provided by
```

Comment:
```suggestion
	 * <code>getProject(resource)</code> method provided by
```
It's only one method, right?